api/src/main/resources/workbench-api.yaml

```diff
@@ -4125,13 +4125,13 @@ definitions:
     type: string
     description: a domain for concepts corresponding to a table in the OMOP schema
     enum:
-    - OBSERVATION
-    - PROCEDURE
-    - DRUG
     - CONDITION
-    - MEASUREMENT
-    - DEVICE
     - DEATH
+    - DEVICE
+    - DRUG
+    - MEASUREMENT
+    - OBSERVATION
+    - PROCEDURE
```

Comment:
nit: why did we shuffle these?

Reply:
So that I can test and force user to add new Enums to the end of the list :-) see  StorageEnumsTest#test_PrePackagedConceptSetsToFromStorageAreInOrdinalOrder and StorageEnumsTest#test_DomainToFromStorageAreInOrdinalOrder. These tests will break if you change order in the yaml file.